tools/benchmark/cmd/put.go

```diff
@@ -134,21 +134,21 @@ func compactKV(clients []*v3.Client) {
 		ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
 		resp, err := c.KV.Get(ctx, "foo")
 		cancel()
-		if err != nil {
```

Comment:
why change these? the usual pattern is check for an error with an if and have the non-error path as fall-through